### PR TITLE
Feature/gechina 94

### PR DESCRIPTION
**Pull request by develaper**
https://vizzuality.atlassian.net/browse/GECHINA-94?atlOrigin=eyJpIjoiMDJlYmRhODdmMGU5NGYwZTk0MWRmZTE4NTQyMGRhMWIiLCJwIjoiaiJ9

Adds remaining fields to groups, subgroups and indicators:

- widget_list.
- slug.
- default subgroup.
- categories.
- category filters.
- default_visualization.

### CODE REVIEW

File: app/controllers/api/v1/groups.rb [hunk 1]
```diff
@@ -5,8 +5,11 @@ class Groups < Grape::API
 
 			resource :groups do
 				desc "Return all groups"
+				params do
+					use :pagination
+				end
 				get "", root: :groups do
-					present Group.all, with: API::V1::Entities::Group
+					present Group.page(params[:page]).per(params[:per_page]), with: API::V1::Entities::Group
```
Comment (agnessa) [inline app/controllers/api/v1/groups.rb:12]: for all the endpoints with pagination we need to enforce `ORDER BY`

**Review by agnessa — CHANGES_REQUESTED** [review]
I think I only see an issue with missing `ORDER BY` in paginated endpoints. Other wise it is good, just advance warning that it seems it might require more work if it is merged after https://github.com/Vizzuality/gefc-api/pull/13, so let's coordinate.

**Comment by develaper** [comment]
Yes! I was planning to merge [yours](https://github.com/Vizzuality/gefc-api/pull/13/files) and then adapt a few things to make it match and also use the FindOrCreateEntity in the widgets importer.

File: app/models/group.rb [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 
-    def set_slug
-        self.slug = name.downcase.gsub(/[[:space:]]/, '-')
-    end
-
+    # Returns subgroup name if exist or nil
+    #
     def default_subgroup
-        subgroups&.first&.name
+        subgroup&.name_en
```
Comment (agnessa) [inline app/models/group.rb:15]: this one is returned in the payload, so it should be locale-specific rather than _en I think?

**Review by agnessa — APPROVED** [review]

**Pull request merged**